lib/dialects/postgres/query.js

```diff
@@ -20,11 +20,20 @@ class Query extends AbstractQuery {
    * @private
    */
   static formatBindParameters(sql, values, dialect) {
-    let bindParam = [];
+    const stringReplaceFunc = value => {
+      if (_.isString(value)) {
```

Comment:
Nit: You can make this `value => typeof value === 'string' ? value.replace(...) : value`

Reply:
I thought that wasn’t a complete check due to the issues explained at https://stackoverflow.com/questions/4059147/check-if-a-variable-is-a-string-in-javascript

Reply:
Most of the codebase already uses typeof checks, https://stackoverflow.com/questions/5750656/whats-the-point-of-new-stringx-in-javascript explains the debacle pretty well.

In the end, sequelize doesn't really support these boxed types and I don't think it should as it would increase complexity in hundreds of places.